.github/workflows/release.yml

```diff
@@ -60,6 +60,7 @@ jobs:
           BRANCH: gh-pages
           FOLDER: build/docs
           VERSION: ${{ steps.release_version.outputs.release_version }}
+          TARGET_REPOSITORY: ${{ github.repository == 'micronaut-projects/micronaut-core' && env.docsRepository || github.repository }}
```

Comment:
I'm not familiar with this syntax, but it looks like it would return a `boolean`?

Reply:
This is the trick: https://github.com/actions/runner/issues/409#issuecomment-752775072

Reply:
Thanks, so there are worse things than yaml in the world :)